server/server_test.go

```diff
@@ -101,13 +129,54 @@ var (
 			},
 		},
 	}
+
+	RuleContentInternal1 = ics_content.RuleContent{
+		Summary:    testdata.Rule1.Summary,
+		Reason:     testdata.Rule1.Reason,
+		Resolution: testdata.Rule1.Resolution,
+		MoreInfo:   testdata.Rule1.MoreInfo,
+		Plugin: ics_content.RulePluginInfo{
+			Name:         testdata.Rule1.Name,
+			NodeID:       "",
+			ProductCode:  "",
+			PythonModule: internalTestRuleModule,
+		},
+		ErrorKeys: map[string]ics_content.RuleErrorKeyContent{
+			"ek1": {
+				Generic: testdata.RuleErrorKey1.Generic,
+				Metadata: ics_content.ErrorKeyMetadata{
+					Condition:   testdata.RuleErrorKey1.Condition,
+					Description: testdata.RuleErrorKey1.Description,
+					Impact:      testdata.ImpactIntToStr[testdata.RuleErrorKey1.Impact],
+					Likelihood:  testdata.RuleErrorKey1.Likelihood,
+					PublishDate: testdata.RuleErrorKey1.PublishDate.UTC().Format(time.RFC3339),
+					Tags:        testdata.RuleErrorKey1.Tags,
+					Status:      "active",
+				},
+			},
+		},
+	}
```

Comment:
Move this definition to `testdata` package in `insights-results-aggregator-data` repository, but it can be done later